docs/src/components/home/GetStartedButtons.tsx

```diff
@@ -6,16 +6,28 @@ import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRou
 import ContentCopyRounded from '@mui/icons-material/ContentCopyRounded';
 import CheckRounded from '@mui/icons-material/CheckRounded';
 import Link from 'docs/src/modules/components/Link';
+import NpmCopyButton from 'docs/src/components/action/NpmCopyButton';
 
 interface GetStartedButtonsProps extends BoxProps {
-  callToAction?: string;
+  primaryCTA?: string;
+  primaryCTAto: string;
+  secondaryCTA?: string;
+  secondaryCTAto?: string;
   installation?: string;
-  to: string;
+  installationAlternative?: string;
 }
 
 export default function GetStartedButtons(props: GetStartedButtonsProps) {
   const [copied, setCopied] = React.useState(false);
-  const { installation, callToAction = 'Get started', to, ...other } = props;
+  const {
+    primaryCTA = 'Get started',
+    primaryCTAto,
+    secondaryCTA,
+    secondaryCTAto,
+    installation,
+    installationAlternative,
```

Comment:
To me, 
`primaryLabel, primaryUrl, primaryUrlTarget, secondaryLabel, secondaryUrl, secondaryUrlTarget, installation, altInstallation` seems like the most understandable props structure 

(`...target` being the props which specifies whether the link's `target` attribute, we would need this on the secondary button for Toolpad to open a Calendly URL in a separate tab)

Reply:
Just pushed a change in this direction! Can you double-check and see if the `target` idea is what you were suggesting?